### PR TITLE
[#2876][Fix]: JSON string cannot be accurately verified

### LINKED ISSUE #2876: [Bug] Unable to correctly verify the Flink cluster URL
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### Java Version

1.8

### Scala Version

2.12.x

### StreamPark Version

2.1.1

### Flink Version

1.14.5

### deploy mode

remote

### What happened

When creating a new Flink cluster, if an incorrect cluster URL is entered, it can also pass the verification. Here is an example of a correct URL demonstration and two error demonstrations:
image
When an incorrect cluster is configured, it can cause trouble for future use.

### Error Exception

```log
no exception
```

### Screenshots

as above

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!(您是否要贡献这个PR?)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Gooch0922**
## What changes were proposed in this pull request

Issue Number: close #2876 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

**Comment by Gooch0922** [comment]
If not modified, there will be issues encountered in the issue where JacksonUtils is used to verify JSON. If additional keys in JSON need to be ignored, they may need to be configured in entity classes, such as
@JsonIgnoreProperties (ignoreUnknown=true)

**Comment by wolfboys** [comment]
Thanks for your contribution, can you give an example or provide the testcase?

**Comment by Gooch0922** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by Gooch0922** [comment]
I copied the same error example in #2876, as shown in the following figure, and correctly passed the testcase in my environment. 
image
I am not sure if this testcase needs to be submitted, as it depends on the existing environment.How to comply with regulations if submission is required.

File: streampark-console/streampark-console-service/src/main/resources/application.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #
 
 server:
-  port: 10000
+  port: 10001
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/resources/application.yml:19]: Why change the port, can you tell us the reason?

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/JacksonUtils.java [hunk 2]
```diff
@@ -38,7 +38,7 @@ private JacksonUtils() {}
   static {
     MAPPER = new ObjectMapper();
     MAPPER.registerModule(new DefaultScalaModule());
-    MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, true);
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/JacksonUtils.java:41]: This method is used in many places, and changing this parameter could potentially cause other bugs. Unless it has undergone thorough testing, can you explain why it is necessary to change this parameter?

**Comment by Gooch0922** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by wolfboys** [comment]
Hi, Hi, this PR has some unexpected changes and has been stagnant for too long. We'll close for now, but we would be very happy if you could submit the PR again

**Pull request closed without merging**